<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QWidget" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>514</width>
    <height>380</height>
=======
    <width>550</width>
    <height>148</height>
>>>>>>> 369c4630d7dd8026acb0db7e96671758f25dbe9b
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log</string>
  </property>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>110</x>
     <y>230</y>
     <width>41</width>
=======
     <x>20</x>
     <y>25</y>
     <width>72</width>
>>>>>>> 369c4630d7dd8026acb0db7e96671758f25dbe9b
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>110</x>
     <y>270</y>
     <width>31</width>
=======
     <x>20</x>
     <y>60</y>
     <width>72</width>
>>>>>>> 369c4630d7dd8026acb0db7e96671758f25dbe9b
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLog">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>110</x>
     <y>310</y>
     <width>93</width>
     <height>51</height>
=======
     <x>20</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
>>>>>>> 369c4630d7dd8026acb0db7e96671758f25dbe9b
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReg">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>240</x>
     <y>310</y>
     <width>93</width>
     <height>51</height>
=======
     <x>120</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
>>>>>>> 369c4630d7dd8026acb0db7e96671758f25dbe9b
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>180</x>
     <y>230</y>
     <width>211</width>
=======
     <x>120</x>
     <y>20</y>
     <width>401</width>
>>>>>>> 369c4630d7dd8026acb0db7e96671758f25dbe9b
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>180</x>
     <y>270</y>
     <width>211</width>
=======
     <x>120</x>
     <y>60</y>
     <width>401</width>
>>>>>>> 369c4630d7dd8026acb0db7e96671758f25dbe9b
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHint">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>360</x>
     <y>310</y>
     <width>71</width>
=======
     <x>220</x>
     <y>100</y>
     <width>301</width>
>>>>>>> 369c4630d7dd8026acb0db7e96671758f25dbe9b
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
